tests/apps/search/test_viewsets_categories.py

```diff
@@ -148,8 +148,9 @@ def test_viewsets_categories_search(self, mock_search):
                             {"term": {"kind": "subjects"}},
                             {
                                 "multi_match": {
-                                    "query": "Science",
+                                    "analyzer": "english",
```

Comment:
I didn't understand what changes in the code lead to this change of behavior?

Reply:
This one here: https://github.com/openfun/richie/pull/1005/files#diff-848b539601c898e16a2f4cc46dbcf50fR319